pandas/tests/frame/test_convert_to.py

```diff
@@ -71,6 +71,10 @@ def test_to_dict_timestamp(self):
         tm.assert_dict_equal(test_data_mixed.to_dict(orient='split'),
                              expected_split_mixed)
 
+    def test_to_dict_index_not_unique_with_index_orient(self):
+        df = DataFrame({'a': [1, 2], 'b': [0.5, 0.75]}, index=['A', 'A'])
```

Comment:
can you add the issue number as a comment

Reply:
Added.